feature/experimental/bgp/ate_tests/bgp_long_lived_graceful_restart/bgp_long_lived_graceful_restart_test.go

```diff
@@ -294,14 +294,17 @@ func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
 	}
 }
 
-func verifyPortsUp(t *testing.T, dev *ondatra.Device) {
+func verifyPortsUp(t *testing.T, dev *ondatra.Device) int {
+	var portDownCnt int
 	t.Helper()
 	for _, p := range dev.Ports() {
 		status := gnmi.Get(t, dev, gnmi.OC().Interface(p.Name()).OperStatus().State())
 		if want := oc.Interface_OperStatus_UP; status != want {
+			portDownCnt++
```

Comment:
It seems that replacing `t.Error()` with `t.Fatal()` is the easier approach if you want to stop the test when a port is not up. No need to introduce additional counters and propagate that state.